packages/main/src/MenuItem.ts

```diff
@@ -223,6 +226,10 @@ class MenuItem extends ListItem {
 		return this.shadowRoot!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
 	}
 
+	get _menuItems() {
```

Comment:
Why are the getters for the same thing in the same change different?  Can we unify them?